msgpack/_unpacker.pyx

```diff
@@ -20,6 +19,7 @@ from cpython.exc cimport PyErr_WarnEx
 cdef extern from "Python.h":
     ctypedef struct PyObject
     cdef int PyObject_AsReadBuffer(object o, const void** buff, Py_ssize_t* buf_len) except -1
+    cdef int PyObject_GetBuffer(object o, Py_buffer *view, int flags) except -1
```

Comment:
Exceptions were ignored (e.g., if array is not C-contiguous), which led to access violations.
Should I move related functions as well?